bin/ci/PSScriptAnalyzerSettings.psd1

```diff
@@ -0,0 +1,3 @@
+@{
+    ExcludeRules=@('PSUseDeclaredVarsMoreThanAssignments')
```

Comment:
any reason why we are excluding this rule?

Reply:
I'll add a comment line - all plans trigger this rule because lines in `plan.ps1` like `pkg_name=7zip` count as being variables that are declared but are "unused".

Reply:
I added a comment section to document the reason rules get excluded.